frontend/src/app/modules/challenges/online-editor/online-editor-page/online-editor-page.component.html

```diff
@@ -15,32 +15,59 @@
                         <button ngbNavLink (click)="selectTab('Submissions')">Submissions</button>
                         <ng-template ngbNavContent>Submissions here</ng-template>
                     </li>
+
+                    <li [ngbNavItem]="'Output'" [class.selectedTab]="isSelected('Output')">
+                        <button ngbNavLink (click)="selectTab('Output')">Output</button>
+                        <ng-template ngbNavContent>Output here</ng-template>
+                    </li>
                 </ul>
                 <div class="content" [ngbNavOutlet]="nav"></div>
             </div>
         </as-split-area>
 
         <as-split-area [size]="50" [minSize]="splitRightMinSize">
-            <div class="editor h-100">
+            <div class="editor h-50">
                 <div class="header">
                     <div class="language-selection">
                         <fa-icon [icon]="'info'" size="sm"></fa-icon>
                         <app-dropdown-select
                             [isMultiSelection]="false"
                             [items]="languages"
                             [selectedItem]="selectedLanguage"
-                            (selectedItemsChanged)="onSelectedLanguageChanged($event)"></app-dropdown-select>
+                            (selectedItemsChanged)="onSelectedLanguageChanged($event)">
+                        </app-dropdown-select>
+                        <app-dropdown-select
+                            [isMultiSelection]="false"
+                            [items]="languageVersions"
+                            [selectedItem]="selectedLanguageVersion">
+                        </app-dropdown-select>
                     </div>
-                    <div class="vertical-divider"></div>
-                    <button>
-                        <span>Auto</span>
-                    </button>
                 </div>
                 <ngx-monaco-editor
```

Comment:
It wasn't done by you, but you can change it to our code-editor component which has monaco-editor with same Inputs